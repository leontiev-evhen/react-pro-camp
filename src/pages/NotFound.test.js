import React from 'react';
import { shallow } from 'enzyme';
import NotFound from './NotFound';

describe('NotFound', () => {
	const Component = shallow(<NotFound />);

	it('NotFound component render correctly', () => {
		expect(Component).toMatchSnapshot();
	});

	it('NotFound text', () => {
		expect(Component.find('h2').text()).toEqual('Page not found');
	});
});
