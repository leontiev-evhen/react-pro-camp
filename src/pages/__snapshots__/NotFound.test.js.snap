// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound NotFound component render correctly 1`] = `
<h2>
  Page not found
</h2>
`;
